cmd/example-client/main.go

```diff
@@ -242,22 +249,23 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	timer := time.NewTimer(1 * time.Minute)
-	defer timer.Stop()
-	go func() {
-		<-timer.C
-		ic <- syscall.SIGABRT
-	}()
+
 	signal.Notify(ic, os.Interrupt, syscall.SIGTERM)
-	for !firstContact {
-		err = c.Publish("scanner/target/cmd/director", cmds.NewCreate("target", "director", ""))
+	for i := 0; i < 10 && !firstContact; i++ {
```

Comment:
What is the `i` supposed to do in this?
I am not sure if this works as intended ...

Reply:
The for loop has two conditions. With the first correct response of the director the client breaks free, as `firstContact` becomes true.